examples/ReinterpretLog.hs

```diff
@@ -179,7 +179,7 @@ instance
         unReinterpretLogC k
 
     R other ->
-      ReinterpretLogC (alg (R (handleCoercible other)))
+      ReinterpretLogC (handleCoercible other)
```

Comment:
This is where I’ve come down re: #235: this is all boilerplate, so let’s at least make it minimal, obvious, & consistent boilerplate.